src/App.js

```diff
@@ -6,9 +6,9 @@ import {nanoid} from 'nanoid';
 export default function App() {
   const [messages, setMessages] = useState(fakeMessages);
 
-  function addNewMessage(newMessage) {
+  function onNewMessage(newMessage) {
```

Comment:
The naming was better the way it was before :) 

When passing a function as prop, it would be passed with the prefix `on` like `onNewMessage` - as you correctly do. The name of your actual function declaration does not start with the prefix `on`, so the better naming for it would be `addNewMessage` - like you had before.